Chronological/Filter.cs

```diff
@@ -192,7 +192,9 @@ private static string MemberExpressionToString(MemberExpression memberExpression
                 return eventFieldMemberExpression.EscapedEventFieldName;
             }
 
-            if (memberExpression.Expression.NodeType == ExpressionType.Constant)
+            if (memberExpression.Expression.NodeType == ExpressionType.Constant
+                || (memberExpression.Expression.NodeType == ExpressionType.MemberAccess &&
```

Comment:
Am I right to assume this means that obj.Prop2 now works but it doesn't cater for obj.Prop2.SubProp1?

It feels like there is a more flexible recursive solution to this rather than just dealing with a single member depth.

Reply:
Yes, you are totally right. I have fixed this problem by adding "GetInnermostExpressionType" method.